src/Microsoft.AspNet.Mvc.Localization/IServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Linq;
+using Microsoft.AspNet.Mvc.Localization;
+using Microsoft.AspNet.Mvc.Razor;
+using Microsoft.Framework.Internal;
+using Microsoft.Framework.WebEncoders;
+
+namespace Microsoft.Framework.DependencyInjection
+{
+    public static class ServiceCollectionExtensions
```

Comment:
This type should have a more specific name, such as `MvcLocalizationServiceCollectionExtensions`.

If this was copied from somewhere else, we should fix that place too. The problem with the current name is that the same namespace+type name would exist in dozens of assemblies, with no easy way to tell them apart.